docs/application/environment.md

```diff
@@ -75,3 +75,9 @@ Release: RRJQTMVKRS
 ```
 
 `convox env edit` allows you to interactively update your environment variables in a terminal editor 😊
+
+<div class="block-callout block-show-callout type-info" markdown="1">
+If you update your env vars through the CLI, that will create a new release containing your changes.  For this to take effect, you will need to promote that release: `$ convox releases promote RRJQTMVKRS`
```

Comment:
Using a specific release ID can be confusing. What about just showing them `convox env set --promote`

Reply:
I'll add that in, as that's super useful, but still think it's good to make a 2-step process clear as well.